src/repo/learnocaml_exercise.ml

```diff
@@ -37,8 +37,8 @@ let encoding =
        (req "test" string)
        (req "solution" string)
        (req "max-score" int)
-       (req  "depend" (option string))
-       (req  "dependencies" (list string)))
+       (opt "depend" (string))
+       (dft "dependencies" (list string) []))
```

Comment:
This looks reasonable to me. 

@lsylvestre, what do you think?

Reply:
I think it is correct. 
Sorry for introducing a bug !